featurebyte/routes/feature_namespace/controller.py

```diff
@@ -110,7 +110,7 @@ async def list(
         )
 
         feature_id_to_primary_table_ids = {}
-        async for feature_dict in self.feature_service.list_documents_iterator(
+        async for feature_dict in self.feature_service.list_documents_as_dict_iterator(
             query_filter={"_id": {"$in": list(default_feature_ids)}}
         ):
             feature = FeatureModel(**feature_dict)
```

Comment:
would be cool to clean up some of these in the future too since this call can now be simplified to use the new `list_documents_iterator` directly!